css/admin-settings.scss

```diff
@@ -1,24 +1,24 @@
 #ldap-write-support-admin-settings {
 
-  .mono {
-    font-family: monospace;
-    font-size: larger;
-  }
+	.mono {
```

Comment:
the server is going away from scss, and for 25 it is necessary to drop it. Not sure, maybe it can be done in the same move, or we take it like this and follow up on the scss stuff later on.

Reply:
Fixed this. As that were only three rules, I simply moved that into the component.